### PR TITLE
Declared license mappings

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

**Comment by sschuberth** [comment]
FYI @blaumeiser-at-bosch, I've compared the license texts and they are indeed plain `BSD-3-Clause`.

File: spdx-utils/src/main/resources/declared-license-mapping.yml [hunk 1]
```diff
@@ -371,6 +371,7 @@
 "The W3C License": W3C
 "The W3C Software License": W3C
 "Three-clause BSD-style": BSD-3-Clause
+"Trilead Library License (BSD-Like)": BSD-3-Clause
```
Comment (tsteenbe) [inline spdx-utils/src/main/resources/declared-license-mapping.yml:375]: See bottom of https://svnkit.com/TRILEAD-LICENSE.txt it not just BSD-3-Clause but BSD-3-Clause AND MIT

Comment (sschuberth) [reply spdx-utils/src/main/resources/declared-license-mapping.yml:375]: Ah, thanks. I overlooked that the code from Bouncy Castle was actually copied and modified. Fixed.

**Comment by sschuberth** [comment]
@tsteenbe I can wait with this until your https://github.com/oss-review-toolkit/ort/pull/2706 is merged.

**Pull request merged**